alacritty_terminal/src/config/window.rs

```diff
@@ -39,6 +39,10 @@ pub struct WindowConfig {
     #[serde(deserialize_with = "from_string_or_deserialize")]
     pub class: Class,
 
+    /// XEmbed parent
+    #[serde(deserialize_with = "failure_default")]
+    pub embed: Option<u64>,
```

Comment:
I think we should only offer this on Linux/BSD.

Reply:
I'm new to all this.  Can you elaborate on what you're expecting based on this coment?

Reply:
You should add `#[cfg(not(any(target_os = "macos", windows)))]` above `#[serde(...)]`. It's expecting that this option will only work on Linux/BSD systems, take a look on `StartupMode`.`SimpleFullscreen` below, as an example.

Reply:
I'm actually not so sure anymore if this is a good idea. Since this is an `Option` anyways, I don't think it would make any difference at all.

The same thing goes for the command line arguments, while it would be nice to only show the commands available on that platform, I don't want to ship multiple manpages etc. So I think the best bet is to just skip that for now. I don't think it's such a big deal.

So let's focus on the error handling first, that's far more important.